.gitignore

```diff
@@ -2,3 +2,5 @@
 cache
 target
 /ontology/mapping.zip
+.DS_Store
```

Comment:
System and IDE specific ignores should not go into the repo. Please use your global .gitignore file.